packages/mdctl-axon-tools/lib/StudyDataTranslations.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable camelcase */
+const globby = require('globby'),
+      path = require('path'),
+      fs = require('fs'),
+      _ = require('lodash'),
+      { privatesAccessor } = require('@medable/mdctl-core-utils/privates'),
+      { Driver } = require('@medable/mdctl-api-driver'),
+      { Org } = require('@medable/mdctl-api-driver/lib/cortex.object'),
+      { Fault } = require('@medable/mdctl-core')
+
+class StudyDataTranslations {
+
+  constructor(client, options = {}) {
+    Object.assign(privatesAccessor(this), {
+      client,
+      options
+    })
+  }
+
+  getOrg() {
+    const { client } = privatesAccessor(this),
+          driver = new Driver(client),
+          org = new Org(driver)
+
+    return org
+  }
+
+  isStudyDataTranslations({ input = process.cwd(), manifest }) {
+    let manifestData = manifest
+    if (!manifestData) {
+      const location = globby.sync(['manifest.{json,yaml}'], { cwd: input })
+      if (location.length > 0 && fs.existsSync(`${input}/${location[0]}`)) {
+        manifestData = JSON.parse(fs.readFileSync(`${input}/${location[0]}`))
+      }
+    }
+
+    if (manifestData.studyDataTranslations && !_.has(manifestData, 'i18ns')) {
+      throw Fault.create('mdctl.kInvalidArgument.missingI18nObjects', {
+        message: 'The manifest is missing "i18ns" objects',
+        reason: 'Incase of using "studyDataTranslations" option. The manifest file must include "i18ns" objects.'
+      })
+    }
+
+    return manifestData.studyDataTranslations
+  }
+
+  async writeStudyDataTranslationsToDisk({ input = process.cwd(), format = 'json' }) {
+    const location = globby.sync(['i18ns/**/*.{json,yaml}'], { cwd: input }),
+          tasks = await this.readAuthenticationTasks(),
```

Comment:
Do we have a way to know whether the study data being imported are Auth task related or not?